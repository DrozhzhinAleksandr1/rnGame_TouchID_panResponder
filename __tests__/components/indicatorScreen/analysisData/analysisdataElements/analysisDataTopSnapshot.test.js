import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { AnalysisDataTop } from '../../../../../app/components/indicatorScreen/analysisData/analysisDataElements/analysisDataTop';

it('create snapshot AnalysisDataTop', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<AnalysisDataTop quotationText={'Котировка:'} quotationNum={'0.94527'} btnText={'покупать'} action={'buy'} />);
    expect(toolbarSnapshot).toMatchSnapshot();
});
