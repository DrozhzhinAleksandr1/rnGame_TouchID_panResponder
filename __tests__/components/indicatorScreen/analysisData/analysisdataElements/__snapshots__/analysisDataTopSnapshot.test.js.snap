// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot AnalysisDataTop 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#1D2E3E",
      "flexDirection": "row",
      "height": 48,
      "justifyContent": "space-around",
      "shadowColor": "rgba(0, 0, 0, 0.15)",
      "shadowOffset": Object {
        "height": 4,
        "width": 0,
      },
      "shadowOpacity": 1,
      "shadowRadius": 4,
      "width": "100%",
      "zIndex": 1,
    }
  }
>
  <AnalysisDataTopLeftBtn
    quotationNum="0.94527"
    quotationText="Котировка:"
  />
  <Component
    style={
      Object {
        "borderLeftColor": "#5A6772",
        "borderLeftWidth": 1,
        "height": "100%",
        "width": 0.5,
      }
    }
  />
  <Connect(AnalysisDataTopRightBtn)
    action="buy"
    btnText="покупать"
  />
</Component>
`;
